wwr/about.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="blah blah bleh">
+    <meta name="author" content="Lucas Caldas da Costa">
+
+    <!-- Heading Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&display=swap" rel="stylesheet">
+
+    <!-- Text Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400..900&family=Fira+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <!-- Stylesheet -->
+     <link rel="stylesheet" href="styles/rafting.css">
+
+    <title>White Water Rafting</title>
+</head>
+<body>
+    <header>
+        <img class="logo" src="images/valfenda_wwr_log_true_palette.png" alt="site logo">
+        <nav class="nav_bar">
+            <a href="index.html">Home</a>
+            <a href="about.html">About Us</a>
+            <a href="trips.html">Trips</a>
+            <a href="contact.html">Contact Us</a>
+        </nav>
+    </header>
+    <main>
+        <div class="hero">
+            <img src="images/wwr_1.jpg" alt="background image">
+            <h1>Valfenda Rafting</h1>
+            <article>
+                <img src="images/wwr_8.jpg" alt="happy client/working employee">
+                <p>Purpose; Mission, Creed, Motto, Etc. Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ex, voluptatum!</p>
+            </article>
+        </div>
+        <section>
+            <h2>History</h2>
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Obcaecati nostrum et exercitationem totam iste veritatis molestiae dicta fuga voluptatibus tempore!</p>
+        </section>
+        <section>
+            <h2>Adventure Awaits You!</h2>
+            <aside class="img_smp">
+            <img src="images/wwr_2.jpg" alt="white water rafting 1">
+            <img src="images/wwr_3.jpg" alt="white water rafting 2">
+            <img src="images/wwr_4.jpg" alt="white water rafting 3">
+            <img src="images/wwr_5.jpg" alt="white water rafting 4">
+            <img src="images/wwr_6.jpg" alt="white water rafting 5">
```

Comment:
Each individual image should be nested inside a `figure` tag, along with a `figcaption` that describes the image. For example:

```html
<figure>
  <img src="images/wwr_6.jpg" alt="White water rafting 5">
  <figcaption>White water rafting 5</figcaption>
</figure>
```

Please never forget to read every instruction in the assignments. This is the 8th step of this activity's instructions.

[Activity Instructions](https://byui-cse.github.io/wdd130-ww-course/week04/prove-rafting-enhance.html#:~:text=Style%20the%20Adventures,Day%20Adventure%22%2C%20etc.)

![Image](https://github.com/user-attachments/assets/34c5ba95-361d-41fb-b887-d7296f35a576)